### ISSUE TITLE
"Could not load config file"

### ISSUE THREAD

**Issue by Maverynthia** [dmunozv04/iSponsorBlockTV#199]
Before opening an issue make sure that there are no duplicates and that you are on the latest version.
**Describe the bug**
Used `main_tui` to get a config file, config file isn't loaded when then using `main`
Also says it created a config file and it doesn't

**To Reproduce**
1. use `main_tui.py` get config file
2.  run `main.py`
3.  get error

**Expected behavior**
That it finds the config file and works.

**Screenshots**
![image](https://github.com/user-attachments/assets/f5efe7cc-8940-4acf-af21-9c20e03eb47e)
![image](https://github.com/user-attachments/assets/206046ec-f749-498a-b4ca-8d5cffa5916d)
![image](https://github.com/user-attachments/assets/89d803df-7d3b-45f3-88da-440401ac69ae)
![image](https://github.com/user-attachments/assets/3ff7ac2a-6225-4746-b319-c852c012fb86)

You can see I'm running it from the source `src` directory. TUI puts the config file in that directory.
C:\Python\iSponsorBlockTV-main\src

**iSponsorBlockTV server (please complete the following information):**
Windows 10 64
Python 3.11

**Apple TV (please complete the following information):**
Samsung Series 7

**Comment by Soheab** [comment]
Does the config have a device?

**Comment by dmunozv04** [comment]
It sounds like another instance of this: https://github.com/dmunozv04/iSponsorBlockTV/issues/192#issuecomment-2358185433 
TL;DR: Use main.py --config instead of main_tui.py since it was just made for ease in development. I should probably just remove it

**Comment by dmunozv04** [comment]
Interesting that main --setup doesn't run setup, that must be a bug. Try with `main setup`

**Comment by Maverynthia** [comment]
> Please post your config (making sure to redact sensitive info). Also, calling `main_tui.py` sets the data_dir to a different place that what the regular program does; try opening the configurator by using `main.py --setup`

![image](https://github.com/user-attachments/assets/3eeaa63c-5819-4777-a75e-3ec7a90108a0)

Doesn't look like `--config` is working either. Where is the config.json supposed to go in relation to /src/

![image](https://github.com/user-attachments/assets/d0cfe682-bd71-4ccb-9bc6-9c6e5c980768)
`{
    "devices": [
        {
            "screen_id": "<randomNumbersAndLetters>",
            "name": "Snamsoong s743",
            "offset": 0
        }
    ],
    "apikey": "",
    "skip_categories": [
        "sponsor",
        "selfpromo",
        "interaction",
        "exclusive_access"
    ],
    "channel_whitelist": [],
    "skip_count_tracking": true,
    "mute_ads": false,
    "skip_ads": true,
    "auto_play": false
}`

**Comment by Soheab** [comment]
The config is supposed to go at `/data/config.json/config.json`
With the last one being the actual file.

```
data/
  config.json/
    config.json
src/
    iSponsorBlockTv/
        ...
...
```

**Comment by Soheab** [comment]
The config is supposed to go at `/data/config.json/config.json`
With the last one being the actual file.

```
data/
  config.json/
    config.json
src/
    iSponsorBlockTv/
        ...
...
```
